elastalert/create_index.py

```diff
@@ -49,7 +49,7 @@ def main():
         host = args.host if args.host else raw_input('Enter elasticsearch host: ')
         port = args.port if args.port else int(raw_input('Enter elasticsearch port: '))
         use_ssl = (args.ssl if args.ssl is not None
-                   else args.no_ssl if args.no_ssl is not None
+                   else not args.no_ssl if args.no_ssl is not None
                    else raw_input('Use SSL? t/f: ').lower() in ('t', 'true'))
```

Comment:
I'd use:
`
parser.add_argument('--ssl', action='store_true', default=None, help='Use SSL')
parser.add_argument('--no-ssl', dest='ssl', action='store_false', help='Do not use SSL')
`